### DISCUSSION TITLE
Subdomain configuration not working for client with existing Vercel project

### DISCUSSION THREAD

**Discussion by schicaelmott** [vercel/community#3182] [Help]
### Summary

My company creates websites. We host websites on our clients' subdomains. 

Usually, I just add my client's subdomain to my Vercel project, and then my client will add a CNAME record (as instructed by Vercel) to their DNS records (usually Google Domains or GoDaddy). 

However, one of my clients already has their own Vercel project. So, when I add the domain `subdomain.client.com` to my Vercel project, I see the following message: 

`Another Vercel account is using this domain. Please set the following TXT record on _vercel.client.com to use shop.client.com in this project. Once the verification is completed and the domain is successfully configured, the TXT record can be removed.`

My client added the TXT record within GoDaddy over 4 days ago. However, Vercel hasn't told me that the domain is verified for use with my account. 

Does the TXT record need to go in my client's Vercel account? I'm confused as to how to set this up.

### Steps to Reproduce

All details in the summary

**Comment by smaeda-ks** [comment]
Looks like you already submitted a support ticket for this. Please check my last reply in 00145114
